classes/submission/maps/Schema.php

```diff
@@ -421,16 +421,31 @@ protected function mapByProperties(array $props, Submission $submission, bool|Co
      */
     protected function getPropertyReviewAssignments(Enumerable $reviewAssignments, bool|Collection $anonymizeReviews = false): array
     {
+        $request = Application::get()->getRequest();
+        $currentUser = $request->getUser();
+        $currentUserRoleIds = array_map(
+            function (Role $role) {
+                return $role->getId();
+            },
+            $currentUser->getRoles($this->context->getId())
+        );
+
+        // is the user a manager, sub-editor, or site admin?
+        $canSeeAllReviewAssignments = (
+            in_array(Role::ROLE_ID_MANAGER, $currentUserRoleIds) ||
+            in_array(Role::ROLE_ID_SUB_EDITOR, $currentUserRoleIds) ||
```

Comment:
We shouldn't include here submission editors not assigned to the submission. And probably managers and admins assigned into the submission in the author role or reviewer role. Or assigned to the other stage than the review. 

We already calculated roles in the stages property: https://github.com/pkp/pkp-lib/blob/aac75ee578cc834b8d2cd99cd07071ba5098a45a/schemas/submission.json#L253

Using it you can actually check if the user has those roles, it includes global ones and those related to the assignment.  

And also you can use `array_intersect()` instead of `in_array`.